src/main/java/com/example/tour/TourApplication.java

```diff
@@ -19,13 +21,16 @@ public static void main(String[] args) {
     }
 
     @Bean
-CommandLineRunner commandLineRunner(PlaceRepository placeRepository, ActivityRepository activityRepository){
+CommandLineRunner commandLineRunner(PlaceRepository placeRepository, ActivityRepository activityRepository, RatingRepository ratingRepository){
```

Comment:
commandLineRunner generates new entries on the database each time we run the project. It'd be nice to have it commented and use it when necessary. For final testing purposes, we'll need to implement JUnit tests